src/main/java/Duke.java

```diff
@@ -1,10 +1,244 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    /**
+     * Runs the Duke program.
+     */
+    public static void main(String[] args) throws FileNotFoundException {
+        ArrayList<Task> lists = new ArrayList<Task>();
```

Comment:
Perhaps you can rename lists to taskList, since it is supposed to be singular? Additionally, taskList can be declared as List<Task> instead of ArrayList<Task>.